### ISSUE TITLE
Test Checkout Flow With Tabbing

### ISSUE THREAD

**Issue by alexfinnarn** [CUCentralAdvancement/giving-frontend#111]
For accessibility, you should be able to get through the site without a mouse or finger...actually, I don't know how that would work with a touch screen.

I tried tabbing through things briefly and it did not go well.

**Comment by alexfinnarn** [comment]
added to cu-giving tests.
